vs060:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100  

  # Position - Joint Position Trajectory Controllers -------------------
  arm_position_trajectory_controller:
    type: "position_controllers/JointTrajectoryController"
    joints:
      - j1
      - j2
      - j3
      - j4
      - j5
      - flange
    constraints:
      goal_time: 2.0                  # Defaults to zero
      stopped_velocity_tolerance: 0.1 # Defaults to 0.01
      joint1:
        trajectory: 0
        goal: 0.2
      joint2:
        trajectory: 1
        goal: 0.2
      joint3:
        trajectory: 0
        goal: 0.2
      joint4:
        trajectory: 0
        goal: 0.2
      joint5:
        trajectory: 0
        goal: 0.2
      joint6:
        trajectory: 1
        goal: 0.2
  
    state_publish_rate:  50 # Defaults to 50
    action_monitor_rate: 20 # Defaults to 20

  hand_position_trajectory_controller:
    type: "position_controllers/JointTrajectoryController"
    joints:
      - H1_F1J1
      - H1_F1J2
      - H1_F1J3
      - H1_F2J1
      - H1_F2J2
      - H1_F2J3
      - H1_F3J1
      - H1_F3J2
      - H1_F3J3

# hand:
#   position_trajectory_controller:
#   type: "position_controllers/JointTrajectoryController"
#   joints:
#     - H1_F1J1
#     - H1_F1J2
#     - H1_F1J3
#     - H1_F2J1
#     - H1_F2J2
#     - H1_F2J3
#     - H1_F3J1
#     - H1_F3J2
#     - H1_F3J3